style.css

```diff
@@ -164,19 +164,19 @@ nav li a:hover{
 
 .left a{
     display:inline-block;
-    background-color:#ceeddb;
+    background-color:#f2e5cf;

```

Comment:
Version 3's colour scheme consists of sandy colours to suit the website's theme of skincare better. Borders and hover functions are of light browns/beige tones to complement the sandy tones.